App/Sagas/TextileSagas.js

```diff
@@ -114,11 +114,6 @@ export function * handleNewAppState ({previousState, newState}) {
 }
 
 export function * triggerCreateNode () {
-  const locked = yield select(IpfsNodeSelectors.locked)
```

Comment:
Seems like if we're going to remove this use of `lock`, we should remove it everywhere.